MekHQ/src/mekhq/campaign/parts/Part.java

```diff
@@ -1233,6 +1236,10 @@ public boolean isReservedForReplacement() {
         return reserveId != null;
     }
 
+    public boolean isUsedForRefitPlanning() { return usedForRefitPlanning; }
+
+    public void setUsedForRefitPlanning(boolean flag) { usedForRefitPlanning = flag; }
```

Comment:
These should be split into three lines, the IDE will automatically shrink them to one when desired and the difference is better in terms of style.

Reply:
Sorry about that. I forgot about IntelliJs habit of collapsing the one line methods.